tests/document/test_instance.py

```diff
@@ -3332,6 +3332,38 @@ class Embedded(EmbeddedDocument):
         f1.ref  # Dereferences lazily
         self.assertEqual(f1, f2)
 
+    def test_embedded_document_equality_with_lazy_ref(self):
+        class Job(EmbeddedDocument):
+            boss = LazyReferenceField("Person")
```

Comment:
Could you add a similar test case (unless there is one already), but using dbref=True `boss = LazyReferenceField("Person", dbref=True)` ? Since most of the code would be similar, feel free to simply add a `boss_dbref = LazyReferenceField("Person", dbref=True)` in your Job class

Reply:
Done. This exposed another issue, fixed in 329f030a.

Reply:
:+1: